externs/shaka/manifest_parser.js

```diff
@@ -140,8 +140,9 @@ shaka.extern.ManifestParser = class {
  *   getBandwidthEstimate: function():number,
  *   onMetadata: function(string, number, ?number,
  *                        !Array.<shaka.extern.MetadataFrame>),
- *   disableStream: function(!shaka.extern.Stream),
- *   addFont: function(string, string)
+*    closeSegmentIndex: function(shaka.extern.Stream, function()),
+*    disableStream: function(!shaka.extern.Stream),
+*    addFont: function(string, string)
```

Comment:
Please fix the indentation:

```suggestion
 *    closeSegmentIndex: function(shaka.extern.Stream, function()),
 *    disableStream: function(!shaka.extern.Stream),
 *    addFont: function(string, string)
```